src/screen.c

```diff
@@ -42,7 +42,7 @@ void screen_print_debug(const char* message, int duration)
     char print[100];
     snprintf(print, sizeof(print), "%s", message);
     UG_ClearBuffer();
-    UG_FontSelect(&font_font_a_9X9);
+    UG_FontSelect(&font_font_a_11X10);
```

Comment:
What about [screen_init](https://github.com/digitalbitbox/bitbox02-firmware/blob/267b9c240d28cfd682bb2708e1aa178f79411736/src/screen.c#L113)?

Reply:
changed it. note the new commit msg:

> Unclear if this has any real effect in the firmware, as most
components that print text select the font again locally.

 > The bootloader uses the same init, and the strings are affected by
this. For now we force it back to 9x9 in the bootloader, as the
'upgrading' label looks more appealing in the original size.

Reply:
ah, right. keeping 9X9 for the bootloader makes sense.